modules/network-observability-enriched-flows.adoc

```diff
@@ -37,11 +38,29 @@ spec:
         targetHost: "ipfix-collector.ipfix.svc.cluster.local"
         targetPort: 4739
         transport: tcp or udp           <4>
-
-
+ -  type: OpenTelemetry                 <3>
+      openTelemetry:
+        targetHost: my-otelcol-collector-headless.otlp.svc
+        targetPort: 4317
+        type: grpc                      <5>
+        logs:                           <6>
+          enable: true
+        metrics:                        <7>
+          enable: true
+          prefix: netobserv
+          pushTimeInterval: 20s         <8>
+          expiryTime: 2m
+        fieldsMapping:                  <9>
+          input: source.address
+          output: SrcAddr
```

Comment:
Since this is optionnal, should we put it in the example ?

The risk here is someone copy / pasting the example without reading the note

Reply:
I decided to add the OTEL in with what we've already got for Exporting enriches network flow data. I experimented with breaking them up, but I felt like ultimately adding it in here isn't so bad, especially since we'll add that column to the existing flow formats table and that info will just be a link. 

Maybe here, I can # out the fieldsMapping so if it is copied, its at least commented?

Reply:
yeah that will work ! Thanks Sara